### PR TITLE
Feature/sploit 935 | SQL server TLS version plugin

### PR DESCRIPTION
**Pull request by AkhtarAmir**

### CODE REVIEW

File: collectors/aws/collector.js [hunk 1]
```diff
@@ -812,6 +816,13 @@ var postcalls = [
                 filterValue: 'FunctionArn'
             }
         },
+        RDS: {
```
Comment (giorod3) [inline collectors/aws/collector.js:819]: does this need to be a custom collector call?

File: plugins/aws/rds/sqlServerTLSVersion.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+                
+                if (!parameters) return paramcb();
+
+                if (parameters.err || !parameters.data){
+                    helpers.addResult(results, 3,
+                        'Unable to query for parameters: ' + helpers.addError(parameters));
+                    return paramcb();
+                }
+
+                if (!parameters.data.Parameters || !parameters.data.Parameters.length) {
```
Comment (giorod3) [inline plugins/aws/rds/sqlServerTLSVersion.js:52]: you can add the DB Parameter Group ARN here as the resource for the result to be more descriptive

File: plugins/aws/rds/sqlServerTLSVersion.spec.js [hunk 3]
```diff
@@ -0,0 +1,201 @@
+var expect = require('chai').expect;
+var sqlServerTLSVersion = require('./sqlServerTLSVersion.js');
+
+const parameterGroups = [
+    {
+        "DBParameterGroupName": "default.sqlserver-ex-14.0",
+        "DBParameterGroupFamily": "sqlserver-ex-14.0",
+        "Description": "Default parameter group for sqlserver-ex-14.0",
+        "DBParameterGroupArn": "arn:aws:rds:us-east-1:560213429563:pg:default.sqlserver-ex-14.0"
```
Comment (giorod3) [inline plugins/aws/rds/sqlServerTLSVersion.spec.js:9]: is this your account number?

Comment (AkhtarAmir) [reply plugins/aws/rds/sqlServerTLSVersion.spec.js:9]: Yes

File: plugins/aws/rds/sqlServerTLSVersion.js (hunk 2 above)
Comment (AkhtarAmir) [reply plugins/aws/rds/sqlServerTLSVersion.js:52]: Okay

File: collectors/aws/collector.js (hunk 1 above)
Comment (AkhtarAmir) [reply collectors/aws/collector.js:819]: No. I had confusion earlier but you clerified in the meeting and I have removed the custom collector for this postcall.

**Pull request merged**